DigitalLearningSolutions.Web/Scripts/trackingSystem/reports.ts

```diff
@@ -1,7 +1,15 @@
 import Chartist from 'chartist';
 import getPathForEndpoint from '../common';
 
-showMoreRows(false);
+const path = getPathForEndpoint('TrackingSystem/Centre/Reports/Data');
+const activityToggleRowClass = 'js-toggleable-activity-row';
+const activityToggleableRowDisplayNone = 'none';
+const activityToggleableRowDisplayTableRow = 'table-row';
+const request = new XMLHttpRequest();
+const viewMoreLink = <HTMLElement>document.getElementsByClassName('js-toggle-row-button').item(0);

```

Comment:
It's worth extracting the class name to make it clear what should match with _ActivityTable.cshtml. It would be nice to have a comment here pointing to the cshtml as well.